src/components/Header/Header.jsx

```diff
@@ -0,0 +1,41 @@
+import { Link, useHistory } from 'react-router-dom';
+import { useUser } from '../../hooks/useUser';
+import { signOutUser, signInUser } from '../../services/users';
+import styles from './Header.css';
+export default function Header() {
+  const { user, setUser } = useUser();
+  const history = useHistory();
+
+  function logOutButton() {
+    signOutUser();
+    setUser('');
+    history.push('/');
+  }
+  console.log(user);
```

Comment:
it's okay to leave `console.log` lines in your code, but there should still be an empty line here for readability